docs/paper/dev/misc/debugging.md

```diff
@@ -0,0 +1,77 @@
+# Debugging your plugin
+
+Debugging your plugin is vital to being able to fix bugs and issues in your plugin. This page will cover some of the most common debugging techniques.
+
+### Randomly printing to the console
+
+One of the most common debugging techniques is to print to the console. This is likely something you've done before as its very simple. 
+It has a few downsides though. It can be hard to find the print statements in the console, and it can be hard to remove them all when you're done debugging.
```

Comment:
```suggestion
One of the most common debugging techniques is to print to the console. This is likely something you've done before, as it's very simple. 
This has a few downsides, though. It can be hard to find the print statements in the console, and it can be hard to remove them all when you're done debugging. Most notably, you have to recompile your plugin and restart the server to add or remove debugging.
```